fedimint-load-test-tool/src/common.rs

```diff
@@ -0,0 +1,393 @@
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::time::Duration;
+use std::vec;
+
+use anyhow::{anyhow, bail, Result};
+use bitcoin::secp256k1;
+use devimint::cmd;
+use devimint::util::ToCmdExt;
+use fedimint_client::secret::PlainRootSecretStrategy;
+use fedimint_client::sm::OperationId;
+use fedimint_client::transaction::TransactionBuilder;
+use fedimint_client::{Client, ClientBuilder};
+use fedimint_core::config::ClientConfig;
+use fedimint_core::core::IntoDynInstance;
+use fedimint_core::encoding::Decodable;
+use fedimint_core::module::registry::ModuleDecoderRegistry;
+use fedimint_core::module::CommonModuleGen;
+use fedimint_core::task::TaskGroup;
+use fedimint_core::{Amount, OutPoint, TieredMulti, TieredSummary};
+use fedimint_ln_client::{LightningClientExt, LightningClientGen, LnPayState};
+use fedimint_mint_client::{
+    MintClientExt, MintClientGen, MintClientModule, MintCommonGen, SpendableNote,
+};
+use fedimint_wallet_client::WalletClientGen;
+use futures::StreamExt;
+use lightning_invoice::Invoice;
+use tokio::sync::mpsc;
+use tracing::info;
+
+use crate::MetricEvent;
+
+pub async fn get_notes_cli(amount: &Amount) -> anyhow::Result<TieredMulti<SpendableNote>> {
+    cmd!(FedimintCli, "fetch").out_string().await?;
+    cmd!(FedimintCli, "spend", amount.msats.to_string())
+        .out_json()
+        .await?["note"]
+        .as_str()
+        .map(parse_ecash)
+        .transpose()?
+        .ok_or_else(|| anyhow::anyhow!("missing notes output"))
```

Comment:
`context` (use anyhow::Context) is really neat here
```suggestion
        .context("missing notes output")
```